pacta/build_with_tag.sh

```diff
@@ -76,7 +69,8 @@ do
     rm -rf "$repo"
 done
 
-image_tar_gz="2dii_pacta.tar.gz"
+parent="$(dirname $(which $0))"
```

Comment:
This is how we find the Dockerfile:

The command and Dockerfile need to live in the same directory (we might relax this eventually, by adding an argument `--path`). The special argument `$0` contains the command the user typed -- i.e. `*build_with_tag.sh`. Then we use `which` to get the full path. Finally we extract the dirname (e.g. `~/git/2diidockerrunner/pacta`), and there we now we have the Dockerfile.

Reply:
fancy! 👍